pyproject.toml

```
@@ -63,6 +63,13 @@ dependencies = [
 packages = ["src/dc_custom_component"]
 
 [tool.hatch.envs.code-quality.scripts]
+format = "black src tests --check"
+format-fix = "black src tests"
+lint = "ruff ."
```

I believe it was changed to `ruff check .`
```suggestion
lint = "ruff ."
```

---

thats the lint-ci command below 🙈

---

Don't we usually do `ruff check --output-format=github` for the lint-ci command?

---

oh damn. you are right. it should also be ruff check . (--fix). What i used is deprecated